features/step_definitions/logging.rb

```diff
@@ -180,11 +180,14 @@
     | component=kibana |
   })
   cl.wait_until_kibana_is_ready
-  # lastly check the cronjob.
-  if env.version_cmp('4.5', user: user) < 0
+  # check the cronjob curator.
+  if env.version_cmp('5.0', user: user) < 0
     raise "Failed to find cronjob for curator" if cron_job('curator').schedule.nil?
```

Comment:
How about `raise "Failed to find cronjob for curator" unless cron_job('curator').exists?` ?